### ISSUE TITLE
Link device code to MOD_gpusubsurf.c

### ISSUE THREAD

**Issue by katznboyz1** [katznboyz1/blender-cuda-subdivision-surface-gpu#3]
(Another issue that I'm creating as a reminder for later.)

This article mentions the process of linking .cu files to normal .cpp files, and then it also describes the process of compiling with with nvcc/g++/(hopefully) clang

[(Subsection) Advanced Usage: Using a Different Linker](https://developer.nvidia.com/blog/separate-compilation-linking-cuda-device-code/)

I will work on this later, once I have `gpu-subsurf.cu` completed and it is ready for linking back into `MOD_gpusubsurf.c`.

**Comment by katznboyz1** [comment]
Temporarily closing issue as it appears that the device code will be located in another file than MOD_gpusubsurf.c. This may even lead to the removal of gpu-subsurf.cu and the replacement of it with another file in blenkernel.
